_examples/ceph-source-example.adoc

```diff
@@ -60,15 +73,19 @@ No architecture, this is a connectivity example.
 
 2. Start a Jupyter notebook with the commands:
 
-   oc new-app rimolive/notebook
-   oc expose svc/notebook
+  oc new-app radanalyticsio/base-notebook \
+    -e JUPYTER_NOTEBOOK_PASSWORD=supersecret \
+    -e JUPYTER_NOTEBOOK_X_INCLUDE=https://radanalytics.io/assets/s3-source-example/s3-source-example.ipynb
```

Comment:
instead of adding this notebook, why not just add the notebook listed in step 4?